tasks/run-local.sh

```diff
@@ -281,19 +288,20 @@ test_pr() {
     ./inspect-queue --amqp localhost:5671;"
 
     LOGS_URL="$S3_URL_HOST/logs/"
+    CURL="curl --cacert $SECRETS/ca.pem --silent --fail --show-error"
 
     # wait until the unit-test got run and published, i.e. until the non-chunked raw log file exists
     for retry in $(seq 60); do
-        LOG_MATCH="$(curl --silent --show-error $LOGS_URL| grep -o "pull-${PR}-[[:alnum:]-]*-unit-tests/log<")" && break
+        LOG_MATCH="$($CURL $LOGS_URL| grep -o "pull-${PR}-[[:alnum:]-]*-unit-tests/log<")" && break
```

Comment:
This is begging for a `python3 -m lib.s3 get` subcommand or so.

Reply:
You mean `ls`? But the next few usages of `$CURL` would be a `get` indeed. I'd keep this as a follow-up though, as it needs changes to bots.